src/tests/class_tests/openms/source/MRMFeaturePickerFile_test.cpp

```diff
@@ -0,0 +1,171 @@
+
+  TEST_EQUAL(cgp_list[1].component_group_name, "orn")
+  TEST_EQUAL(cgp_list[1].params.getValue("stop_after_feature"), 5)
+  TEST_REAL_SIMILAR(cgp_list[1].params.getValue("stop_after_intensity_ratio"), 0.0001)
+  TEST_REAL_SIMILAR(cgp_list[1].params.getValue("min_peak_width"), -1.0)
+  TEST_EQUAL(cgp_list[1].params.getValue("peak_integration"), "smoothed")
+  TEST_EQUAL(cgp_list[1].params.getValue("background_subtraction"), "none")
+  TEST_EQUAL(cgp_list[1].params.getValue("recalculate_peaks"), "false")
+  TEST_EQUAL(cgp_list[1].params.getValue("use_precursors"), "false")
+  TEST_REAL_SIMILAR(cgp_list[1].params.getValue("recalculate_peaks_max_z"), 1.0)
+  TEST_REAL_SIMILAR(cgp_list[1].params.getValue("minimal_quality"), -10000.0)
+  TEST_REAL_SIMILAR(cgp_list[1].params.getValue("resample_boundary"), 0.05)
+  TEST_EQUAL(cgp_list[1].params.getValue("compute_peak_quality"), "true")
+  TEST_EQUAL(cgp_list[1].params.getValue("compute_peak_shape_metrics"), "true")
+
+  TEST_EQUAL(cgp_list[3].component_group_name, "ser-L")
+  TEST_EQUAL(cgp_list[3].params.getValue("stop_after_feature"), 5)
+  TEST_REAL_SIMILAR(cgp_list[3].params.getValue("stop_after_intensity_ratio"), 0.0001)
+  TEST_REAL_SIMILAR(cgp_list[3].params.getValue("min_peak_width"), -1.0)
+  TEST_EQUAL(cgp_list[3].params.getValue("peak_integration"), "smoothed")
+  TEST_EQUAL(cgp_list[3].params.getValue("background_subtraction"), "none")
+  TEST_EQUAL(cgp_list[3].params.getValue("recalculate_peaks"), "false")
+  TEST_EQUAL(cgp_list[3].params.getValue("use_precursors"), "false")
+  TEST_REAL_SIMILAR(cgp_list[3].params.getValue("recalculate_peaks_max_z"), 1.0)
+  TEST_REAL_SIMILAR(cgp_list[3].params.getValue("minimal_quality"), -10000.0)
+  TEST_REAL_SIMILAR(cgp_list[3].params.getValue("resample_boundary"), 0.05)
+  TEST_EQUAL(cgp_list[3].params.getValue("compute_peak_quality"), "true")
+  TEST_EQUAL(cgp_list[3].params.getValue("compute_peak_shape_metrics"), "true")
+
+  TEST_EQUAL(cgp_list[4].component_group_name, "group2")
+
+  TEST_REAL_SIMILAR(cgp_list[4].params.getValue("stop_after_intensity_ratio"), 0.0001)
+  TEST_REAL_SIMILAR(cgp_list[4].params.getValue("min_peak_width"), -1.0)
+  TEST_EQUAL(cgp_list[4].params.getValue("peak_integration"), "smoothed")
+  TEST_EQUAL(cgp_list[4].params.getValue("background_subtraction"), "none")
+  TEST_EQUAL(cgp_list[4].params.getValue("recalculate_peaks"), "false")
+  TEST_EQUAL(cgp_list[4].params.getValue("use_precursors"), "false")
+  TEST_REAL_SIMILAR(cgp_list[4].params.getValue("recalculate_peaks_max_z"), 1.0)
+  TEST_REAL_SIMILAR(cgp_list[4].params.getValue("minimal_quality"), -10000.0)
+  TEST_REAL_SIMILAR(cgp_list[4].params.getValue("resample_boundary"), 0.05)
+  TEST_EQUAL(cgp_list[4].params.getValue("compute_peak_quality"), "true")
+  TEST_EQUAL(cgp_list[4].params.getValue("compute_peak_shape_metrics"), "true")
+
+  TEST_EQUAL(cgp_list[4].params.exists("stop_after_feature"), false)
+}
+END_SECTION
```

Comment:
When adding in the other method tests, it would be good to change some of the other parameter values to test that each one is being captured and set correctly.  I know this is a problem that I ran into multiple times.